### ISSUE TITLE
Version numbers?

### ISSUE THREAD

**Issue by jbouzanquet** [CSULA-Qdots/filter#9]
Should we label the programs with version numbers, and if so should these be output in log files?

**Comment by blackdjinn** [comment]
Yes to both, I think. I'd already given some thought to putting the GIT commit number and anything else into the log at the start of the run... that way if we wanted to go back to that code snapshot we could.

**Comment by jbouzanquet** [comment]
That would require typing the commit hash into the file, which would cause a new commit, changing the hash? Hashes are stable, but are the numbers stable even when merging branches?

**Comment by blackdjinn** [comment]
The commit numbers should be stable, yes... and no, it wouldn't if all we did was run 'git status' from within the script  and capture the output and append that to the log. It would tell us exactly where we are with it when it was run.

**Comment by jbouzanquet** [comment]
That works if we assume we're always running on linux. It seems like several things work that way, maybe we should just roll with it.

**Comment by blackdjinn** [comment]
I'm gonna hammer on this a bit RSN...
